src/lang/meta.js

```diff
@@ -0,0 +1,35 @@
+const localazyMetadata = {
+  projectUrl: "https://localazy.com/p/chec-ui-library",
+  baseLocale: "en",
+  languages: [
```

Comment:
Does this error if you're missing the ja.js and fr.js files locally? In the dashboard this file is hand-written to only contain English as a language, so people don't have to run Localazy to have their dashboard work.

Reply:
@ScopeyNZ good catch, I overwrote what's supposed to be there while testing.  Restored.